.github/workflows/qa-checks.yml

```
@@ -0,0 +1,23 @@
+name: Open Source Programs QA Checks
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    name: Linting and Code Formatting Checks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Installing QA-CHECKS Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install black
+          pip install isort
+          pip install flake8
+      - name: Running QA-CHECKS
+        run: |
+          ./osp-qa-checks
```

You can write it in a single line
```suggestion
        run: ./osp-qa-checks
```